aws/rust-runtime/aws-config/src/profile/timeout_config.rs

```diff
@@ -29,18 +35,15 @@ const PROFILE_VAR_API_CALL_TIMEOUT: &str = "api_call_timeout";
 /// **Sets timeouts for the `default` profile**
 /// ```ini
 /// [default]
-/// connect_timeout = 1.0
-/// read_timeout = 1.0
-/// tls_negotiation_timeout = 0.5
 /// api_call_attempt_timeout = 2
 /// api_call_timeout = 3
 /// ```
 ///
-/// **Sets the `connect_timeout` to 0.5 seconds _if and only if_ the `other` profile is selected.**
+/// **Sets the `api_call_attempt_timeout` to 0.5 seconds _if and only if_ the `other` profile is selected.**
 ///
 /// ```ini
 /// [profile other]
-/// connect_timeout = 0.5
+/// api_call_attempt_timeout = 0.5
```

Comment:
Was changing the profile format intentional?

Reply:
no, like the environment change, this was me being foolish. Good catch

Reply:
Yes, because we don't support timeouts other than the API ones right now. You can set connect timeouts manually in some cases (credentials fetching IIRC) but it doesn't set those timeouts from the profile rn